<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>841</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="Expert">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>651</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Expert_2">
      <attribute name="title">
       <string>Expert</string>
      </attribute>
      <widget class="QListWidget" name="listPlantillas">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>151</width>
         <height>461</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 127);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Templates</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listRules">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>151</width>
         <height>461</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 127);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Rules</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listFacts">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>80</y>
         <width>151</width>
         <height>461</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Facts</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btCargar">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>560</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Cargar clp</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_basura">
       <property name="geometry">
        <rect>
         <x>282</x>
         <y>561</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="User">
      <attribute name="title">
       <string>User</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QPushButton" name="bt_Prueba">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_prueba">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>220</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
